hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -188,6 +295,43 @@ protected String defaultWarehouseLocation(TableIdentifier tableIdentifier) {
         tableIdentifier.name());
   }
 
+  public  Map<String, String> getMetafrpmhiveDb(Database database) {
```

Comment:
We generally omit "get" from method names because it is generic and describes anything. How about renaming this to `convertToMetadata`? That would fix the typos as well.